[2024-02-03T04:20:57.208+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-03T03:00:00+00:00 [queued]>
[2024-02-03T04:20:57.236+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-03T03:00:00+00:00 [queued]>
[2024-02-03T04:20:57.237+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-03T04:20:57.283+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): get_weather_data> on 2024-02-03 03:00:00+00:00
[2024-02-03T04:20:57.290+0000] {standard_task_runner.py:60} INFO - Started process 6742 to run task
[2024-02-03T04:20:57.316+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'get_weather_data', 'scheduled__2024-02-03T03:00:00+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp472eiu02']
[2024-02-03T04:20:57.319+0000] {standard_task_runner.py:88} INFO - Job 186: Subtask get_weather_data
[2024-02-03T04:20:57.393+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.get_weather_data scheduled__2024-02-03T03:00:00+00:00 [running]> on host 324edfad8c3f
[2024-02-03T04:20:57.591+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='get_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-03T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-03T03:00:00+00:00'
[2024-02-03T04:20:57.844+0000] {logging_mixin.py:188} INFO - https://api.openweathermap.org/data/2.5/weather?q=bangkok&appid=***&units=metric
[2024-02-03T04:20:57.844+0000] {logging_mixin.py:188} INFO - {'coord': {'lon': 100.5167, 'lat': 13.75}, 'weather': [{'id': 803, 'main': 'Clouds', 'description': 'broken clouds', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 31.1, 'feels_like': 38.1, 'temp_min': 30.51, 'temp_max': 34.39, 'pressure': 1014, 'humidity': 75, 'sea_level': 1014, 'grnd_level': 1013}, 'visibility': 10000, 'wind': {'speed': 4.12, 'deg': 185, 'gust': 4.19}, 'clouds': {'all': 81}, 'dt': 1706933490, 'sys': {'type': 2, 'id': 2079808, 'country': 'TH', 'sunrise': 1706917497, 'sunset': 1706959097}, 'timezone': 25200, 'id': 1609350, 'name': 'Bangkok', 'cod': 200}
[2024-02-03T04:20:57.846+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-03T04:20:57.867+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=get_weather_data, execution_date=20240203T030000, start_date=20240203T042057, end_date=20240203T042057
[2024-02-03T04:20:57.927+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-03T04:20:57.950+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
